tlatools/org.lamport.tlatools/src/tla2sany/configuration/ConfigConstants.java

```diff
@@ -73,141 +59,10 @@ interface ConfigConstants {
-    "synonym =< \\leq\n" +
-    "operator > 50 50 None infix\n" +
-    "operator \\geq 50 50 None infix\n" +
-    "synonym >= \\geq\n" +
-    //\n" +
-    "operator \\times 100 130 Left nfix\n" +
-    "synonym  \\X \\times\n" +
-    "operator \\ 80 80 None infix\n" +
-    "operator \\intersect 80 80 Left infix\n" +
-    "synonym \\cap \\intersect \n" +
-    "operator \\union 80 80 Left infix\n" +
-    "synonym \\cup \\union\n" +
-    //\n" +
-    "operator ... 90 90 None infix\n" +
-    "operator .. 90 90 None infix\n" +
-    "operator | 100 110 Left infix\n" +
-    "operator || 100 110 Left infix\n" +
-    "operator && 130 130 Left infix\n" +
-    "operator & 130 130 Left infix\n" +
-    "operator $$ 90 130 Left infix\n" +
-    "operator $ 90 130 Left infix\n" +  
-    "operator ?? 90 130 Left infix\n" +
-    //"operator ? 90 130 Left infix\n" +  // Removed requested by Leslie (16 Feb. 01)
-    "operator %% 100 110 Left infix\n" +
-    "operator % 100 110 None infix\n" +
-    "synonym \\mod %\n" +
-    "operator ## 90 130 Left infix\n" +
-    "operator ++ 100 100 Left infix\n" +
-    "operator -- 110 110 Left infix\n" +
-    "operator ** 130 130 Left infix\n" +
-    "operator // 130 130 None infix\n" +
-    "operator ^^ 140 140 None infix\n" +
-    "operator @@ 60 60 Left infix\n" +
-    "operator !! 90 130 None infix\n" +
-    "operator |- 50 50 None infix\n" +
-    "operator |= 50 50 None infix\n" +
-    "operator -| 50 50 None infix\n" +
-    "operator =| 50 50 None infix\n" +
-    "operator <: 70 70 None infix\n" +
-    "operator :> 70 70 None infix\n" +
-    "operator := 50 50 None infix\n" +
-    "operator ::= 50 50 None infix\n" +
-// \n" +
-    "operator \\oplus 100 100 Left infix\n" +
-    "synonym (+) \\oplus\n" +
-    "operator \\ominus 110 110 Left infix\n" +
-    "synonym (-) \\ominus\n" +
-    "operator \\odot 130 130 Left infix\n" +
-    "synonym (.) \\odot\n" +
-    "operator \\oslash 130 130 None infix\n" +
-    "synonym (/) \\oslash\n" +
-    "operator \\otimes 130 130 Left infix\n" +
-    "synonym (\\X) \\otimes\n" +
-// \n" +
-    "operator \\uplus 90 130 Left infix\n" +
-    "operator \\sqcap 90 130 Left infix\n" +
-    "operator \\sqcup 90 130 Left infix\n" +
-    "operator \\div 130 130 None infix\n" +
-    "operator \\wr 90 140 None infix\n" +
-    "operator \\star 130 130 Left infix\n" +
-    "operator \\o 130 130 Left infix\n" +
-    "synonym  \\circ \\o \n" +
-    "operator \\bigcirc 130 130 Left infix\n" +
-    "operator \\bullet 130 130 Left infix\n" +
-    "operator \\prec 50 50 None infix\n" +
-    "operator \\succ 50 50 None infix\n" +
-    "operator \\preceq 50 50 None infix\n" +
-    "operator \\succeq 50 50 None infix\n" +
-    "operator \\sim 50 50 None infix\n" +
-    "operator \\simeq 50 50 None infix\n" +
-    "operator \\ll 50 50 None infix\n" +
-    "operator \\gg 50 50 None infix\n" +
-    "operator \\asymp 50 50 None infix\n" +
-    "operator \\subset 50 50 None infix\n" + // subseteq is builtin
```

Comment:
Let's keep this comment.

Reply:
Deal, but if these two are your only requested changes I would like to make them in the unicode changes just to avoid the merge conflict headache